launcher.py

```
@@ -781,15 +934,15 @@ def main():
         print("=" * 60, flush=True)
         print("", flush=True)
         
+        # 服务已就绪，释放启动锁。
+        release_startup_lock()
+
         # 持续运行，监控服务器状态
```

Ruff (0.15.1)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)